src/ParseFile.js

```diff
@@ -425,6 +444,11 @@ const DefaultController = {
 
   _setXHR(xhr: any) {
     XHR = xhr;
+  },
+
+  deleteFile: function(name, options) {
```

Comment:
Can you write an integration test?

Reply:
I'm writing a test now and running into some blocks which might be the reason why this was never added in the first place. According to the REST API documentation you have to use a DELETE method with the X-Parse-Application-Id and X-Parse-Master-Key headers. The normal RestController.request() function is setting all of the Auth data via the body and changing everything to a POST request... I believe once this hits the server it never goes anywhere... I'm still digging...

Reply:
You will have to use RestController.ajax(). I can add it if you want.

Reply:
I had to step away from my computer. If you can add it, please do!